services/app/src/actions/ServerStatus.test.tsx

```diff
@@ -1,44 +1,44 @@
 import {newMockStore} from '../Testing';
-import {handleAnnouncements} from './Announcement';
+import {handleServerStatus} from './ServerStatus';
 
-describe('Announcement set action', () => {
+describe('ServerStatus set action', () => {
   // Entirely glue code; no testing needed right now.
 });
 
-describe('Fetch Announcements', () => {
-  test('Fetches and sets announcements', () => {
+describe('Fetch ServerStatus', () => {
+  test('Fetches and sets ServerStatus', () => {
```

Comment:
See [here](https://github.com/ExpeditionRPG/expedition/blob/4ccc2e5b39a47b6c1a8756b727db7038e7d66b0d/services/app/src/actions/Web.test.tsx) for an example of testing using `fetchMock`.